osidb/models.py

```diff
@@ -705,6 +705,13 @@ def __str__(self):
         """convert to string"""
         return str(self.uuid)
 
+    def save(self, *args, **kwargs):
+        # Automatically convert empty string values to null, since validation is skipped for
+        # empty strings
+        if self.cve_id == "":
+            self.cve_id = None
+        super().save(*args, **kwargs)
```

Comment:
We usually do these things via `signals.py`. Would it be possible to move it there? I am not saying you have to do it, just asking.

Reply:
Actually for converting None (NULL) values for Char based fields, for data which come from Bugzilla, I've used mixin and it was done in the clean function, so I have no strong preference on this one.